runtime/gc_base/ContinuationObjectList.cpp

```diff
@@ -62,3 +62,18 @@ MM_ContinuationObjectList::addAll(MM_EnvironmentBase* env, j9object_t head, j9ob
 	MM_GCExtensions *extensions = MM_GCExtensions::getExtensions(env);
 	extensions->accessBarrier->setContinuationLink(tail, previousHead);
 }
+
+void
+MM_ContinuationObjectList::cleanupContinuationObject(MM_EnvironmentBase* env, j9object_t objectPtr)
```

Comment:
now this has context (class scope) we can drop Continuation from the name
we can also add Native in the name, but up to you. for example:
releaseNativesForObject